hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -613,6 +614,16 @@ private FlinkOptions() {
       .defaultValue(128)
       .withDescription("Sort memory in MB, default 128MB");
 
+  @AdvancedConfig
+  public static final ConfigOption<String> PARTITION_OVERWRITE_MODE = ConfigOptions
+      .key("partition.overwrite.mode")
```

Comment:
I saw the spark option name is `hoodie.datasource.overwrite.mode`, maybe we should name it `write.partition.overwrite.mode` because it should be a write operation mode.